lib/open_pgp/secret_key_packet.ex

```diff
@@ -128,12 +128,13 @@ defmodule OpenPGP.SecretKeyPacket do
   @type t :: %__MODULE__{
           public_key: PublicKeyPacket.t(),
           s2k_usage: {0..255, binary()},
-          s2k_specifier: S2KSpecifier.t(),
+          s2k_specifier: S2KSpecifier.t() | nil,
           sym_key_algo: OpenPGP.Util.sym_algo_tuple(),
-          sym_key_initial_vector: binary(),
-          sym_key_size: non_neg_integer(),
+          sym_key_initial_vector: binary() | nil,
+          sym_key_size: non_neg_integer() | nil,
           secret_key_material: tuple() | nil,
-          ciphertext: binary()
+          ciphertext: binary(),
+          secret_key_material: tuple()
```

Comment:
seems like this is duplicate key of line 135

Reply:
Oh shoot :). I'll fix that